### PR TITLE
[PL-51352] Delegate 83001 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-51352](https://harness.atlassian.net/browse/PL-51352)

## Description

* Please describe your changes: Delegate 24.05.83001 RN
* Jira/GitHub Issue numbers (if any): [PL-51352](https://github.com/harness/developer-hub/tree/PL-51352)
* Preview links/images (Internal contributors only): https://664cad80095de604f984daad--harness-developer.netlify.app/release-notes/delegate#version-240583001-

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-51352]: https://harness.atlassian.net/browse/PL-51352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -49,6 +49,12 @@ For more information, go to [Delegate expiration policy](/docs/platform/delegate
 
 ## May 2024
 
+### Version 24.05.83000 <!--  May 21, 2024 -->
+
+#### Fixed issues
+
+Added support for proxies via Secure Connect for Github App connectors. (CI-12130, ZD-61883)
```
[the remark this answers is not in the document]
Comment (saifeemustafaqharness) [reply release-notes/delegate.md]: I was unaware that Douglas will be looking over CI RN. So I went to the ticket, and used the RN present on their JIRA ticket.

[the remark this answers is not in the document]
Comment (brian-f-harness) [reply release-notes/delegate.md]: @saifeemustafaqharness Yes, @douglas-j-bothwell is covering CI RN. Please work with him on RN approval for CI and to ensure this item is included on the CI side with the standard delegate dependency verbiage.

**Review by brian-f-harness — APPROVED** [review]

File: release-notes/delegate.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply release-notes/delegate.md]: @saifeemustafaqharness @brian-f-harness I'm tracking 1.27.4 release notes in https://github.com/harness/developer-hub/pull/6621

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Review by pratmit — COMMENTED** [review]
LGTM

**Pull request merged**